### ISSUE TITLE
Non-Controlled Source indirect Text

### ISSUE THREAD

**Issue by benchris** [adelyte/crescendo#56]
For sources like an Airport Express where there is not any control, where would I be able to enter in text that will show up on that touchpanel page if that source is selected?  Something like " Please connect to Airport Express on your device".

Thanks,
Ben

**Comment by adelyte-austin** [comment]
Send any amount of indirect text through the Serial_101 inputs on the source module, and they will populate on the matching T-01__Source_Serial_101_Is strings once connected to the source.

If you examine the Auxiliary page in the graphics, you'll see 2 buttons. One for Sonos (includes app launch), and a generic alert to "Use the manufacturer's remote". These buttons are made visible by sending a 1 through the source digital inputs 101 & 102. A new object for AirPlay could be designed at joined 103.

In summary, use digital visibility, analog modes, indirect text, or even a new unique AirPlay page to achieve the look you're going for.

**Comment by benchris** [comment]
Thanks for the info!
